spec/views/catalog/show.html.erb_spec.rb

```diff
@@ -9,34 +9,34 @@
 
   context 'when entries describe a scanned resource published using an ARK', js: true do
     it 'renders a viewer' do
-      visit '/catalog/4609321'
+      visit '/catalog/9946093213506421'
       expect(page).to have_selector('div#view')
     end
   end
 
   context 'when entries describe a scanned map published using an ARK', js: true do
     it 'renders a viewer' do
-      visit 'catalog/6109323'
-      expect(page).to have_selector('div#view')
+      visit 'catalog/9961093233506421'
+      expect(page).to have_selector('#view')
     end
   end
 
   context 'when entries describe resources published using multiple ARKs', js: true do
     it 'renders multiple viewers' do
-      visit '/catalog/3943643'
+      visit '/catalog/9970446223506421'
```

Comment:
Question: Why is this not 993943643350421?

Reply:
id 993943643350421 Does not exist in Alma. I used the record with the 2 viewers.